### DISCUSSION TITLE
Adding a Blower to PoolController

### DISCUSSION THREAD

**Discussion by spifftek70** [tagyoureit/nodejs-poolController#601] [General]
I am adding more to my automation.  Including a Silence Blower (1.0 hp 120 volt 5.0 amps 60 cycle).  Even though it's only 110v, I'm figuring that at 1HP, it'll need to use a 3HP Relay like the pumps?  It was hooked up to a Intermatic wall switch timer.  There's a purple 110v wire, a white neutral, a ground, and a red 110v wire that is hot only when the timer was turned on.  The blower had a purple (hot) wire, a white neutral, and ground coming from it.

How do you wire up this type of blower for the PoolController?

<img width="1970" alt="Screen Shot 2022-09-10 at 8 01 06 AM" src="https://user-images.githubusercontent.com/11727190/189489187-fa8fdfb0-31c2-4173-a03a-1bdead024ba1.png">

**Comment by rstrouse** [comment]
Yes you will need to use a 3hp relay for that.  The 4 terminals across the bottom are designed to switch two loads.  220/240v split phase current in the united states is done with two 110/120v hots.  In your diagram that would be the red and purple wires.   

So to use 120v you simply use either the red or purple wire and the white (neutral) wire.  So you connect the purple from the pool electrical box to the left screw (line) and the the purple from the blower to the screw (load) right next to it.  Then connect the white wire from the blower to the neutral bus.

I assume you have a breaker panel in there somewhere.  It should have a proper breaker for the load wires on that blower.  If it does not a seized motor can cause a fire due to runaway current.  It would be good to know where the breakers for the pool are installed because spa blowers really do need a dedicated breaker.

The electric code these days also often call out GFCI for applications such as this although it has not always been the case.  It should go from a 120v breaker (this connects to only one hot leg) to the line side of the relay then the load side of the relay to the blower.   Finally the neutral from the blower simply connects to the neutral bus on the pool breaker panel.  Also, if the blower is not double insulated it will need to be bonded with the pool pump and heater.  Do not forget that step if it is set up to be bonded.

**Reply by spifftek70** [reply]
Yeah, I have an entire sub panel dedicated to the pool equipment.  Where are the bonding instructions?

**Reply by rstrouse** [reply]
That depends on the blower.  If it is double insulated it likely doesn’t require bonding.  If it is not then it will.  The instructions for the blower should contain the bonding instructions.  Typically it will have a copper bonding lug on it if it is required.

**Reply by rock-crusher** [reply]
Are you going to abandon the Intermatic wall switch timer, or still plan to use it in conjunction with the automation system?

As for bonding: Somewhere around your pool equipment pad there should be a #8 solid bare copper wire emerging out from the earth. This is your equipotential bonding wire. This conductor loops around externally connecting your pool equipment such as your pump motor, heater, blower etc. by means of an external copper bonding lug as rstrouse mentioned. Hopefully this wire is still fully intact and has not been severed over the years.

**Reply by spifftek70** [reply]
I would very much like to use the timer in conjunction.  How??

There IS a solid bare copper wire connected to the pumps, but not the blower

**Reply by rstrouse** [reply]
That blower is probably double insulated but It wouldn’t be the first time I saw a missing bond.  Helped a buddy of mine replace his equipment and noticed the bond cut off flush with the equipment pad.

Why would you ever want to use that timer?  You have unlimited schedules and egg timers in njsPC.

**Reply by spifftek70** [reply]
True, but we list our pool on Swimply so clients would find the timer easier. That’s probably true for the blower as well.

So if I understand this right, bonding is just grounding the equipment together in a chain? Where do you change the blower? I don’t see a grounding point on the outside

**Reply by rock-crusher** [reply]
> True, but we list our pool on Swimply so clients would find the timer easier. That’s probably true for the blower as well. 

If you’re dead set on using the wall switch timer also, and it’s just for the blower, then I suppose you could do something like this to make it work.

![timer](https://user-images.githubusercontent.com/94593954/189509231-ea21549e-6110-4686-bd6e-1edf72dbdba7.jpg)

But why, you will now have full control of it via your cell phone/tablet. You will be able to quickly set the parameters for your guest’s right from your phone.

> So if I understand this right, bonding is just grounding the equipment together in a chain? Where do you change the blower? I don’t see a grounding point on the outside

Like rstrouse said, that blower is probably double insulated. Some are designed with plastic enclosures and do not require external bonding and yes, the bonding of the equipment usually is daisy chained together.

**Reply by spifftek70** [reply]
Thanks!  Same question, but for a single speed 220V pump. 

<img width="1591" alt="Screen Shot 2022-09-11 at 5 55 47 AM" src="https://user-images.githubusercontent.com/11727190/189528716-e81187f7-5144-4c97-b364-60c6509dfb1e.png">

**Reply by rock-crusher** [reply]
Here is one simple way to do it using basically what is already there.
There are many different ways to achieve what you want.
Take note that the added relay has a 240v coil.
Perhaps someone else may have a more sophisticated way by using the RPI

![Pump Timer](https://user-images.githubusercontent.com/94593954/189547183-a14cfd4f-ba39-4ab1-a786-d60f1220289b.jpg)

**Comment by tagyoureit** [comment]
Random thought and brainstorm... let's say you could give your guests a dumbed down/locked down njsPC interface where they couldn't change any schedules or get into the config, etc but they could turn on/off a couple circuits from their smartphone.  You might post a QR code next to your pool entrance (or send it in email) and your guests could use a temporary pin and have a drop dead simple phone app instead of flipping a switch.  Would that be appealing at all?  Or would you still stick with the physical controls?

**Reply by spifftek70** [reply]
That would be wonderful. I do worry about people trying to change the schedule or configuration settings.

The current page should be accessible by designated admins only.  The link to this dashboard should be:
`http://localhost:5051/admin`
This "dumbed down/locked" page should be the `index` page

**Reply by johnny2678** [reply]
This would be great for Airbnb hosts

**Reply by spifftek70** [reply]
Agreed

**Reply by rock-crusher** [reply]
> Random thought and brainstorm... let's say you could give your guests a dumbed down/locked down njsPC interface where they couldn't change any schedules or get into the config, etc but they could turn on/off a couple circuits from their smartphone. You might post a QR code next to your pool entrance (or send it in email) and your guests could use a temporary pin and have a drop dead simple phone app instead of flipping a switch. Would that be appealing at all? Or would you still stick with the physical controls?

tagyoureit, now that is innovative brainstorming!
Talk about taking this to another level, OMG!
This would open all sorts of possibilities; I can’t wait to see if this comes to fruition.

**Reply by spifftek70** [reply]
are you working on this idea now?

**Reply by tagyoureit** [reply]
It's on a longer list of items I want to get to but I haven't started working on it yet. If you have specific thoughts on what you might like to see you can add them to the [wish list](https://github.com/tagyoureit/nodejs-poolController/discussions/610).
